apps/cli-daemon/src/app/workspace/workspace.controller.ts

```diff
@@ -60,68 +55,45 @@ export class WorkspaceController {
     return { path: this.sessionService.cwd };
   }
 
-  @Get()
+  @Get('project-names')
   getWorkspaceProjectNames(): Promise<string[]> {
     return this.workspaceService.readWorkspaceProjectNames();
   }
 
-  @Get(':projectName')
+  @Get('project/:projectName')
   async getProject(
     @Param('projectName') projectName: string
-  ): Promise<ProjectDefinition> {
-    let project: ProjectDefinition | undefined;
-
-    try {
-      project = await this.workspaceService.readWorkspaceProject(projectName);
-    } catch (err) {
-      this.logger.error(err);
-
-      throw new InternalServerErrorException();
-    }
-
-    if (!project) {
-      throw new NotFoundException();
-    }
+  ): Promise<ProjectDto> {
+    return this.workspaceService.readWorkspaceProject(projectName);
+  }
 
-    return project;
+  @Patch('project/:projectName')
```

Comment:
Patch? it's like Put?

Reply:
yeah but the client can send "put" right?

Reply:
no, why?